### PR TITLE
Add endpoint to disable modifications

### PR DESCRIPTION
**Pull request by klesaulnier**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 1]
```diff
@@ -1138,6 +1138,19 @@ public ResponseEntity<Void> stashNetworkModifications(@Parameter(description = "
         return ResponseEntity.ok().build();
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modifications", params = "active")
+    @Operation(summary = "Update 'active' value for a network modifications for a node")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The network modifications were stashed / restored "), @ApiResponse(responseCode = "404", description = "The study/node is not found")})
```
Comment (antoinebhs) [inline src/main/java/org/gridsuite/study/server/StudyController.java:1143]: Misleading description

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 2]
```diff
@@ -1431,6 +1431,23 @@ public void stashNetworkModifications(UUID studyUuid, UUID nodeUuid, List<UUID>
         notificationService.emitElementUpdated(studyUuid, userId);
     }
 
+    @Transactional
+    public void updateNetworkModificationsActivation(UUID studyUuid, UUID nodeUuid, List<UUID> modificationsUuids, String userId, boolean active) {
+        List<UUID> childrenUuids = networkModificationTreeService.getChildren(nodeUuid);
+        notificationService.emitStartModificationEquipmentNotification(studyUuid, nodeUuid, childrenUuids, NotificationService.MODIFICATIONS_STASHING_IN_PROGRESS);
```
Comment (antoinebhs) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:1437]: is it intended to use MODIFICATIONS_STASHING_IN_PROGRESS? Don't we want a separate notification?

**Review by antoinebhs — COMMENTED** [review]
First comments on reading the code

File: src/main/java/org/gridsuite/study/server/StudyController.java (hunk 1 above)
Comment (klesaulnier) [reply src/main/java/org/gridsuite/study/server/StudyController.java:1144]: fixed

File: src/main/java/org/gridsuite/study/server/service/StudyService.java (hunk 2 above)
Comment (klesaulnier) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java]: Changed to MODIFICATIONS_UPDATING_IN_PROGRESS
I don't think we need a new type of notification

**Review by souissimai — APPROVED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Review by antoinebhs — COMMENTED** [review]
Code OK, some typos in tests

**Pull request merged**